src/docpage.js

```diff
@@ -10,13 +10,131 @@ export default class DocPage extends React.Component {
     if(! doc) return <p>loading...</p>
 
+    let files = doc.data.files && doc.data.files.map((item, index) => {
+      return (
+        <tr key={index}>
+          <td>
+            {item.id ?
+              <a href={`./${item.id}`}>{item.filename}</a> :
+              <span>{item.filename}</span>
+            }
+          </td>
+          <td className="text-muted">{item.content_type}</td>
+          <td>
+            {item.id ?
+              <a href={`./${item.id}/raw/`} title="Original file">
+                <i className="fa fa-file-o"></i>
+              </a> :
+              <code>-- broken link --</code>}
+          </td>
+        </tr>
+      )
+    })
+
+
     return (
-      <pre><code>{JSON.stringify(doc, null, 2)}</code></pre>
+      <div className="card doc-page">
+        <div className="card-block">
+          <p className="card-title lead">#{doc.id}: {doc.data.filename}</p>
+        </div>
+        <p className="bg-faded text-uppercase doc-section-title">
+          Meta</p>
+
+        <div className="card-block">
+          <table className="table table-sm">
+            <tbody>
+              <tr>
+                <td>Path</td>
+                <td><code>{doc.data.path}</code></td>
+              </tr>
+              <tr>
+                <td>Filename</td>
+                <td><code>{doc.data.filename}</code></td>
+              </tr>
+              {doc.data.type &&
+                <tr>
+                  <td>Type</td>
+                  <td><code>{doc.data.type}</code></td>
+                </tr>
+              }
+              {doc.data.type != 'folder' && doc.data.md5 &&
+                <tr>
+                  <td>MD5</td>
+                  <td><code>{doc.data.md5}</code></td>
+                </tr>
+              }
+              {doc.data.type != 'folder' && doc.data.sha1 &&
+                <tr>
+                  <td>SHA1</td>
+                  <td><code>{doc.data.sha1}</code></td>
+                </tr>
+              }
+              {doc.data.lang &&
+                <tr>
+                  <td>Language</td>
+                  <td><code>{doc.data.lang}</code></td>
+                </tr>
+              }
+              {['pdf', 'doc'].includes(doc.data.type) &&
+                <tr>
+                  <td>Created</td>
+                  <td>{doc.data['date-created']}</td>
+                </tr>
+              }
+              {['pdf', 'doc'].includes(doc.data.type) &&
+                <tr>
+                  <td>Modified</td>
+                  <td>{doc.data.date}</td>
+                </tr>
+              }
+            </tbody>
+          </table>
+
+          {doc.data.type == 'email' && (
+            <table className="table table-sm">
+              <tbody>
+                <tr>
+                  <td>From</td>
+                  <td>{doc.data.from}</td>
+                </tr>
+                <tr>
+                  <td>To</td>
+                  <td>{doc.data.to}</td>
```

Comment:
The 'to' field is a list, so this concatenates the e-mail addresses without a separator.
Here's the [snoop implementation](https://github.com/hoover/snoop/blob/e194134c37e13fb2c238ab95d264b549e121df48/snoop/templates/document.html#L149) for using `, ` as a separator.